snuba/datasets/processors/querylog_processor.py

```diff
@@ -70,15 +75,16 @@ def __extract_query_list(
             # ``Cache.get_readthrough`` query execution path. See GH-902.
             is_duplicate.append(int(query["stats"].get("is_duplicate") or 0))
             consistent.append(int(query["stats"].get("consistent") or 0))
-            profile = query.get("profile") or {
+            fallback_profile: ClickhouseQueryProfile = {
                 "time_range": 0,
                 "all_columns": [],
                 "multi_level_condition": False,
                 "where_profile": {"columns": [], "mapping_cols": []},
                 "groupby_cols": [],
                 "array_join_cols": [],
-                "bytes_scanned": 0,
```

Comment:
Why was this removed?

Reply:
because we do not need it later. it also seems that we have not encountered this value in prod when the dict is present

Reply:
I believe the plan was to add this value, but the migrations necessary didn't happen. We can always add it back I guess.

Reply:
bytes scanned is extracted from a different subkey later on, so i have no idea if this was even correct to include in the first place